docs/configuration.md

```diff
@@ -100,6 +100,7 @@ Mandatory parameters are marked as **Required**, which means that they are requi
 | `trailing_only_offset_is_reached` | Only apply trailing stoploss when the offset is reached. [stoploss documentation](stoploss.md). [Strategy Override](#parameters-in-the-strategy). <br>*Defaults to `false`.*  <br> **Datatype:** Boolean
 | `fee` | Fee used during backtesting / dry-runs. Should normally not be configured, which has freqtrade fall back to the exchange default fee. Set as ratio (e.g. 0.001 = 0.1%). Fee is applied twice for each trade, once when buying, once when selling. <br> **Datatype:** Float (as ratio)
 | `trading_mode` | Specifies if you want to trade regularly, trade with leverage, or trade contracts whose prices are derived from matching cryptocurrency prices. [leverage documentation](leverage.md). <br>*Defaults to `"spot"`.*  <br> **Datatype:** String
+| `can_short` | Specifies if the bot can do futures shorting trade. <br>[Strategy Override](#parameters-in-the-strategy) <br>*Defaults to `false`.*  <br> **Datatype:** Boolean
```

Comment:
This parameter should be treated similar to `INTERFACE_VERSION`.

neither of the 2 can be set via the configuration (intentionally) - which is why i didn't add it to this part of the docs (as it's not really set via the configuration).

It's also documented in the strategy part, so i'm not sure if it makes sense to have it elsewhere.